docs/embedding/static-embedding.md

```diff
@@ -8,9 +8,9 @@ redirect_from:
 
 Also known as: static embedding.
 
-In general, embedding works by displaying a Metabase URL inside an iframe on your website. A **signed embed** (or static embed) is an iframe that's secured with a signed JSON Web Token (JWT). The signed JWT will prevent website visitors from poking around in your Metabase through the iframe.
+In general, embedding works by displaying a Metabase URL inside an iframe on your website. A **static embed** (or signed embed) is an iframe that's secured with a signed JSON Web Token (JWT). The signed JWT will prevent website visitors from poking around in your Metabase through the iframe.
```

Comment:
>The signed JWT will prevent website visitors from poking around in your Metabase through the iframe.

I think this is inaccurate. The JWT covers multiple functions:

* authorize loading the artifact, e.g. dashboard - without the JWT in the URL the request will be denied by Metabase- this will work inside or outside of the iframe
* identify the ID of the resource to load in the iframe, e.g. the dashboard ID
* supply values for locked parameters 

My suggestion (wording would need to be revised):

```suggestion
In general, embedding works by displaying a Metabase URL inside an iframe on your website. A **static embed** (or signed embed) is an iframe that's loading a Metabase URL secured with a signed JSON Web Token (JWT). The URL will only load if a JWT is provided and that JWT is signed with the shared secret.

The JWT will also include a reference to the resource to load, e.g. the dashboard ID, and any values for locked parameters.
```